### PR TITLE
info: Don't format size when poking at details

### PR DESCRIPTION
**Pull request by matthiasclasen**
When the -s option is used, it is likely that the result is getting screenscraped and reparsed.
    
Make that easier by producing a plain number in this case.

I just wrote some code in a control-center patch that would benefit from this.

### CODE REVIEW

File: app/flatpak-builtins-info.c [hunk 1]
```diff
@@ -344,7 +344,7 @@ flatpak_builtin_info (int argc, char **argv, GCancellable *cancellable, GError *
       if (opt_show_size)
         {
           maybe_print_space (&first);
-          g_print ("%s", formatted);
+          g_print ("%lu", size);
```
Comment (mwleeds) [inline app/flatpak-builtins-info.c]: Is `guint64` guaranteed to be `unsigned long` on all platforms, or should we use `G_GUINT64_FORMAT`?

Comment (matthiasclasen) [reply app/flatpak-builtins-info.c]: We could, I guess. Although I'm pretty sure we use %lu in other places, and it probably doesn't matter for the platforms flatpak runs on.

Comment (mwleeds) [reply app/flatpak-builtins-info.c]: `git grep "%lu"` doesn't seem to return anything

**Comment by mwleeds** [comment]
@rh-atomic-bot r+ 01de135

**Pull request closed without merging**